tico/utils/convert.py

```diff
@@ -287,6 +287,12 @@ def convert(
     strict: bool = True,
     config: CompileConfigBase = get_default_config(),
 ) -> CircleModel:
+    if mod.training:
```

Comment:
Hmm.. the other day I met this model that didn't have `.eval()` method. Maybe it could happen with `.training` method. How about check if it has `.training` first?